app/routes/__index/certificate/index.tsx

```diff
@@ -16,7 +16,8 @@ export default function CertificateIndexRoute() {
   const certificate: Certificate = {
     id: 1,
     username: 'user_test',
-    subject: `*.user_test.example.com`,
+    domain: `user_test.example.com`,
```

Comment:
This one actually confuses me a bit, from my understanding we'll have one certificate per user. If that was the case, do we have to display all available domains, or do the `*`